src/internal/modules/charge-information/controllers/view-charge-information.js

```diff
@@ -88,11 +88,16 @@ const postReviewChargeInformation = async (request, h) => {
     if (request.payload.reviewOutcome === 'approve') {
       await services.water.chargeVersions.postCreateFromWorkflow(request.params.chargeVersionWorkflowId);
     } else {
+      const { user_id: userId, user_name: userName } = get(request, 'defra.user');
+      const patchObject = {
+        status: request.payload.reviewOutcome,
+        approverComments: request.payload.reviewerComments,
+        chargeVersion: {},
+        createdBy: { id: userId, email: userName }
```

Comment:
similar to my previous comment - this was not the intended business logic. Not sure if this has changed

Reply:
You're right. This was a misunderstanding on my part. I have removed this.